Nos/Views/Modifiers/View+StyledBorder.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import SwiftUI
+
+extension View {
+    func withStyledBorder() -> some View {
+        modifier(StyledBorder())
+    }
+}
+
+struct StyledBorder: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .overlay(
+                RoundedRectangle(cornerRadius: 16)
+                    .strokeBorder(
+                        LinearGradient(
+                            gradient: Gradient(colors: [
+                                Color("styled-border-gradient-top"),
+                                Color("styled-border-gradient-bottom")
```

Comment:
We use SwiftGen to generate color constants from the assets; you should be able to do something like this (but check my casing and spelling):

```suggestion
                                .styledBorderGradientTop,
                                .styledBorderGradientBottom
```